Source/WebCore/platform/graphics/BitmapImage.cpp

```diff
@@ -139,7 +139,7 @@ void BitmapImage::drawPattern(GraphicsContext& context, const FloatRect& destina
         return;
     }
 
-    auto buffer = context.createAlignedImageBuffer(expandedIntSize(tileRect.size()));
```

Comment:
General code organization comment: the `!context.drawLuminanceMask()` early return means that the rest of this function is about luminance masks, but it's easy to miss this on a skim. It would be more readable if we added a new `BitmapImage::drawLuminanceMaskPattern()` and moved this code into it.